src/http.rs

```diff
@@ -12,16 +12,23 @@ mod surf;
 pub struct HTTP<C: HttpClient> {
     authorization_header: C::HeaderName,
     app_name_header: C::HeaderName,
+    x_app_name_header: C::HeaderName,
+    x_sdk_header: C::HeaderName,
+    x_connection_id_header: C::HeaderName,
     instance_id_header: C::HeaderName,
     app_name: String,
+    sdk_version: &'static str,
     instance_id: String,
+    connection_id: String,
```

Comment:
This is being used as a client id, not per-connection
```suggestion
    client_id: String,
```

Reply:
We have an established convention across all SDKs so I'd prefer to have one name for this. We call it connection_id because we're also adding streaming support and we want to treat a long lived SSE stream and interval/polling connections from the same instance as one logical connection.

Reply:
Added a comment explaining the rationale for this name